### ISSUE TITLE
Failed to initialize ORM engine during 1.0.2 -> 1.5.0-rc1 upgrade (no such column: is_fsck_enabled)

### ISSUE THREAD

**Issue by t3hG04T** [go-gitea/gitea#4451]
- Gitea version (or commit ref): ```1.5.0-rc1```
- Git version: ```2.18.0```
- Operating system: ```FreeBSD 11``` (jail in ```FreeNAS 11.1```)
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [x] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [ ] No
  - [x] Not relevant
- Log gist:

```
2018/07/16 21:54:13 [I] Log Mode: File(Info)
2018/07/16 21:54:13 [I] XORM Log Mode: File(Info)
2018/07/16 21:54:13 [I] Cache Service Enabled
2018/07/16 21:54:13 [I] Session Service Enabled
2018/07/16 21:54:13 [I] Mail Service Enabled
2018/07/16 21:54:13 [I] Migration: add tags to releases and sync existing repositories
2018/07/16 21:54:13 [...itea/routers/init.go:60 GlobalInit()] [E] Failed to initialize ORM engine: migrate: do migrate: select repos [offset: 0]: no such column: is_fsck_enabled
```
## Description

Hello good people of gitea! 😄 

I'm currently migrating from ```gogs``` and had successfully moved to ```gitea 1.0.2```.

I've just compiled the binary from ```1.5.0-rc1``` source (checked out the tag) and started the service, but got greeted with the quoted log message (and insta crash).

Looking here https://github.com/go-gitea/gitea/blob/master/models/migrations/migrations.go seems like I'm getting an error during ```v40 -> v41```, but the aforementioned column is only added in step ```v61 -> v62```?

Thank you in advance fo the help!

**Comment by RickZeeland** [comment]
Still getting this error when upgrading from 1.1 to 1.5 on Windows 10 with a PostgreSQL database:
_Failed to initialize ORM engine: migrate: do migrate: select repos [offset: 0]: pq: column "is_fsck_enabled" does not exist_
When installing 1.4 first and then 1.5 everything works.

**Comment by glaszig** [comment]
exactly the same scenario as @RickZeeland. but on freebsd 10.4, installing from ports.

**Comment by t3hG04T** [comment]
@RickZeeland , @glaszig : If you click the linked pull request you can see that the issue will be resolved in the **`1.6` release** (further examining the 1.5.2 code confirms that the fix is not in yet).

If you opt for installing from ports you will have to wait for the release and update, but if building from source you can go ahead and checkout the `v1.6.0-rc1` tag and the problem should be gone :).

**Comment by glaszig** [comment]
@t3hG04T thank you. that clears the situation up for me. meanwhile, since my gitea installation was vanilla anyway, i just deleted the database, installed 1.5 from ports and started all over. used that opportunity to "back port" dark mode as well.
